deploy/roles/sqitch/tasks/main.yml

```diff
@@ -50,7 +50,7 @@
   register: sqitch
   changed_when: '"is up to date" not in sqitch.stdout'
   environment:
-    PATH: "{{ postgres_install_dir|default('/usr/pgsql-9.4') }}/bin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin"
+    PATH: "{{ postgres_install_dir }}/bin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin"
```

Comment:
@rioh Sorry, I was unclear, I meant do the opposite, just hardcode '/usr/pgsql' as the value, since it will always point to `postgres_install_dir` anyways :)

Reply:
Oops,  I see.